lib/iris/cube.py

```diff
@@ -3463,29 +3490,60 @@ def slices(self, ref_to_slice, ordered=True):
             A mix of input types can also be provided. They must all be
             orthogonal (i.e. point to different dimensions).
         ordered : bool, default=True
-            If True, the order which the coords to slice or data_dims
-            are given will be the order in which they represent the data in
-            the resulting cube slices.  If False, the order will follow that of
-            the source cube.  Default is True.
+            If True, subcube dimensions are ordered to match the dimension order
+            in `ref_to_slice`. If False, the order will follow that of
+            the source cube.
 
         Returns
         -------
         An iterator of subcubes.
 
         Examples
         --------
-        For example, to get all 2d longitude/latitude subcubes from a
-        multi-dimensional cube::
-
-            for sub_cube in cube.slices(['longitude', 'latitude']):
-                print(sub_cube)
+        For example, for a cube with dimensions `realization`, `time`, `latitude` and
+        `longitude`:
+
+        >>> fname = iris.sample_data_path('GloSea4', 'ensemble_01[01].pp')
+        >>> cube = iris.load_cube(fname, 'surface_temperature')
+        >>> print(cube.summary(shorten=True))
+        surface_temperature / (K)           (realization: 2; time: 6; latitude: 145; longitude: 192)
+
+        To get all 12x2D longitude/latitude subcubes:
+
+        >>> for sub_cube in cube.slices(['longitude', 'latitude']):
+        ...     print(sub_cube.summary(shorten=True))
+        surface_temperature / (K)           (longitude: 192; latitude: 145)
+        surface_temperature / (K)           (longitude: 192; latitude: 145)
+        surface_temperature / (K)           (longitude: 192; latitude: 145)
+        surface_temperature / (K)           (longitude: 192; latitude: 145)
+        surface_temperature / (K)           (longitude: 192; latitude: 145)
+        surface_temperature / (K)           (longitude: 192; latitude: 145)
+        surface_temperature / (K)           (longitude: 192; latitude: 145)
+        surface_temperature / (K)           (longitude: 192; latitude: 145)
+        surface_temperature / (K)           (longitude: 192; latitude: 145)
+        surface_temperature / (K)           (longitude: 192; latitude: 145)
+        surface_temperature / (K)           (longitude: 192; latitude: 145)
+        surface_temperature / (K)           (longitude: 192; latitude: 145)
+
+
+        .. warning::
+            Note that the dimension order returned in the sub_cubes matches the order specified
+            in the `cube.slices` call, *not* the order of the dimensions in the original cube.
```

Comment:
```suggestion
            in the ``cube.slices`` call, *not* the order of the dimensions in the original cube.
```

Single backticks just render as italics, so the same as the `*not*` shortly afterwards.

Reply:
Fixed